tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -1290,4 +1290,28 @@ class CompletionArgSuite extends BaseCompletionSuite {
     topLines = Some(1)
   )
 
+  check(
+    "inaccessible-defaults",
+    """|/a/src/main/scala/A/package.scala
+       |package object A {
```

Comment:
We don't have support for multiple files in PC tests unfortunately, could you put that or a similar in a single file?

Reply:
Oh darn. Kind of surprising that the test even passed... Updated.

Reply:
I think the compiler might have recovered from the added lines.